openreview/journal/webfield/editorsInChiefWebfield.js

```diff
@@ -0,0 +1,364 @@
+      summary: {
+        id: actionEditor.id,
+        name: actionEditor.name,
+        email: actionEditor.email,
+      },
+      reviewProgressData: {
+        numCompletedReviews: 0,
+        numPapers: 0,
+        papers: [],
+        referrer: referrerUrl
+      },
+      decisionProgressData: {
+        numCompletedMetaReviews: 0,
+        papers: []
+      }
+    };
+  });
+
+  submissions.forEach(function(submission) {
+    if (submission) {
+      var number = submission.number;
+
+      var reviews = submission.details.directReplies.filter(function(reply) {
+        return reply.invitations.indexOf(VENUE_ID + '/Paper' + number + '/-/Review') >= 0;
+      });
+      var decisions = submission.details.directReplies.filter(function(reply) {
+        return reply.invitations.indexOf(VENUE_ID + '/Paper' + number + '/-/Decision') >= 0;
+      });
+      var paperReviewers = reviewersByNumber[number] || [];
+      var paperActionEditors = aeByNumber[number] || [];
+      var paperReviewerStatus = {};
+      var confidences = [];
+      var completedReviews = reviews.length == paperReviewers.length;
+      var formattedSubmission = { id: submission.id, number: number, forum: submission.forum, content: { title: submission.content.title.value, authors: submission.content.authors.value, authorids: submission.content.authorids.value}};
+
+      paperReviewers.forEach(function(reviewer) {
+        var completedReview = reviews.find(function(review) { return review.signatures[0].endsWith('/Reviewer_' + reviewer.anonId); });
+        var status = {};
+        if (completedReview) {
+          var confidenceString = completedReview.content.confidence.value;
+          var confidence = parseInt(confidenceString.substring(0, confidenceString.indexOf(':')));
+          confidences.push(confidence);
+          status = {
+            Recommendation: completedReview.content.recommendation.value,
+            Confidence: confidence,
+            'Review length': completedReview.content.review.value.length
+          }
+        }
+        paperReviewerStatus[reviewer.anonId] = {
+          id: reviewer.id,
+          name: view.prettyId(reviewer.id),
+          email: reviewer.id,
+          completedReview: completedReview && true,
+          forum: submission.id,
+          note: completedReview && completedReview.id,
+          status: status,
+          forumUrl: 'https://openreview.net/forum?' + $.param({
+            id: submission.id,
+            noteId: submission.id,
+            invitationId: VENUE_ID + '/Paper' + number + '/-/Review'
+          })
+        }
+        var reviewerStatus = reviewerStatusById[reviewer.id];
+        if (reviewerStatus) {
+          reviewerStatus.reviewerProgressData.numPapers += 1;
+          reviewerStatus.reviewerStatusData.numPapers += 1;
+          reviewerStatus.reviewerProgressData.papers.push({ note: formattedSubmission, review: completedReview ? { forum: completedReview.forum, status: status } : null});
```

Comment:
opening http://localhost:3030/group?id=.TMLR/Editors_In_Chief as su has an error that completedReview is undefined
![image](https://user-images.githubusercontent.com/60613434/132921535-ef544e76-e6d6-49d8-a96d-1a462f8ea543.png)

Reply:
open the url without logging in has an error in getGroup in webfield-v2 that group is undefined.